package.json

```diff
@@ -171,6 +171,7 @@
     ],
     "modulePaths": [],
     "moduleNameMapper": {
+      "d3$": "<rootDir>/node_modules/d3/dist/d3.min.js",
```

Comment:
d3 exports ES6 natively and jest can't handle it properly. this is an ugly workaround